### ISSUE TITLE
SSL

### ISSUE THREAD

**Issue by UshakovVasilii** [UshakovVasilii/gnome-shell-extension-redmine-issues#2]
Some users said that extension doesn't work with SSL.
I cannot reproduce problem.
I also set `ssl-strict=false` https://github.com/UshakovVasilii/gnome-shell-extension-redmine-issues/commit/d98265817ba6f65368cae446a9370c2148795ca4 but it wasn't helped.
If someone know any details, please let me know.
Thanks!

**Comment by Tab10id** [comment]
I don't think that it still actual but pretty sure that it all about self signed or signed by custom CA certificates. Problem can be solved by adding redmine certificate or their CA to trusted store of OS. I use this extension with ssl and custom CA without any problem.
